clang/test/Driver/sycl.c

```diff
@@ -25,3 +25,6 @@
 // RUN: %clang_cl -### -fsycl -- %s 2>&1 | FileCheck %s --check-prefix=DEFAULT
 
 // DEFAULT: "-sycl-std=2020"
+
+// RUN: %clang_cl -### -fsycl -sycl-std=2017 --no-offloadlib -- %s 2>&1 | FileCheck %s --check-prefix=CHECK-NO-OFFLOADLIB
+// CHECK-NO-OFFLOADLIB-NOT: warning: unknown argument ignored in clang-cl: '--no-offloadlib'
```

Comment:
I probably missed, but is there a corresponding test for clang driver as well?

Reply:
Hi, @srividya-sundaram 
clang and clangxx driver supports this option even without this PR, I add corresponding test to cover clang/clangxx driver to align with other tests in clang/test/Driver/sycl.c.
Thanks very much.